### ISSUE TITLE
Switching between bulleted list and ordered list down a level doesn't work as expected.

### ISSUE THREAD

**Issue by clibu** [bangle-io/bangle.dev#191]
If I have:

- Line 1
   - Line 2 

I should be able to toggle Line 2 between ordered and bulleted list styles and can't. At present the list state is removed.

[Tiptap V2](https://www.tiptap.dev/) works as I would expect.

**Comment by kepta** [comment]
For me it is working in your case, but if I get it to something like
- line 1
  - line 2
    - line 3

And then trigger line 2 to change to ordered list by pressing ctrl-shift-9 it doesn't work.

**Comment by clibu** [comment]
I cleared all content from https://bangle.dev/ and added my example. I can toggle _Line 2_ once but not back again. HTH.

**Comment by lucywang000** [comment]
I can reproduce both cases in the 2nd and 3rd comments above.

**Comment by kepta** [comment]
yes i imagine it to be a separate npm package and not part of the core.
